tests/bluetooth/tester/nrf5340_hci_ipc_cpunet.conf

```
@@ -3,3 +3,4 @@ CONFIG_BT_CTLR_CONN_ISO_LOW_LATENCY_POLICY=y
 CONFIG_BT_CTLR_DATA_LENGTH_MAX=100
 CONFIG_BT_BUF_ACL_RX_SIZE=100
 CONFIG_BT_CTLR_PERIPHERAL_ISO_EARLY_CIG_START=y
+CONFIG_BT_CTLR_ISOAL_SN_STRICT=n
```

Superceded by the below which ensures ISO Data PDUs are enqueued with consecutive seq number.

```suggestion
CONFIG_BT_CTLR_ISOAL_PSN_IGNORE=y
```